addons/sourcemod/configs/scp_sf/classes.cfg

```
@@ -737,15 +737,12 @@
 
 		"items"
 		{
-			"1"	"203"	// Crossvec
+			"1"	"30758"	// Prinny Machete #This guy shouldn't spawn with a gun right at the start. If he can find a gun, good for him, just don't give him one of the best in the mode to fuck people over with.
```

Please leave your opinion out of code comments.
```suggestion
			"1"	"30758"	// Prinny Machete
```